test/crud-with-options.test.js

```diff
@@ -520,6 +521,27 @@ describe('crud-with-options', function() {
       });
     });
   });
+
+  describe('updateAttributes', function() {
+    beforeEach(seed);
+    it('shouldn\'t null not given embedded document properties', function() {
+      if (db.name !== 'mongodb') {
+        console.log('No need to check except for Mongo');
```

Comment:
Please remove this console log. There may be other NoSQL databases besides MongoDB that require this fix.

Reply:
done